<launch>
  <!-- launch file arguments -->
  <arg name="robot_name" default="baxter" />
  
  <!-- the semantic description that corresponds to the URDF -->
  <param name="robot_description_semantic" textfile="$(find baxter_moveit_config)/config/$(arg robot_name).srdf" />
   
  <!-- motion control using ik plugin defined in kinematics.yaml -->
  <node name="baxter_cart_controller" pkg="baxter_teleop" type="cartesian_controller" output="screen">
    <rosparam command="load" file="$(find baxter_moveit_config)/config/kinematics.yaml"/>
    <param name="publish_frequency" value="100"/> <!-- Hz -->
    <param name="planning_group" value="left_arm"/>
  </node>
  
</launch>
